pkg/kubelet/container/helpers.go

```diff
@@ -107,6 +108,14 @@ func HashContainer(container *v1.Container) uint64 {
 	return uint64(hash.Sum32())
 }
 
+// HashAuth - returns a hash code for a CRI pull image auth
+func HashAuth(auth *runtimeapi.AuthConfig) string {
+	hash := fnv.New64a()
```

Comment:
fnv is a non-cryptographic hash algorithm... squinting at whether that matters here, but non-cryptographic + credentials + security make me squint

Reply:
Maybe we switch to sha256 so when we add persistence it is less of a worry.

Reply:
I could add a TODO if you like

Reply:
I think changing to sha256 would be ideal (or even making this configurable to future-proof this?)